content/developer/reference/backend/views.rst

```diff
@@ -1722,6 +1722,33 @@ Possible children of the view element are:
     Enables the 'quick create' and 'drag and drop' features when the kanban
     view is grouped by that field. Default: false.
 
+``header``
+  defines custom :ref:`buttons <reference/views/list/button>` in the control panel that perform an action/call a model's method.
+
+  .. code-block:: xml
+
+      <header>
+          <button name=toDo" type="object" string="Always Display" display="always"/>
+          <button name="toDo" type="object" string="Selection Display"/>
+      </header>
+
+  Does not support any attribute but can have a child:
+
+  .. rst-class:: o-definition-list
+
+  ``button``
+    as a :ref:`button <reference/views/list/button>` which accepts an extra attribute when placed in a `header`:
```

Comment:
ref to list/button

Reply:
it s the same api. We want to duplicate de text ?